### ISSUE TITLE
Unable to launch Spyder - ConnectionResetError: [WinError 995]

### ISSUE THREAD

**Issue by SyedaHinaAli** [spyder-ide/spyder#20524]
### What steps will reproduce the problem?

python 3.8 not working with spyder 5.2.2 so down graded to 5.0.0 but still get this error box popin

### Traceback
```python-traceback
Error on reading from the event loop self pipe
loop: <ProactorEventLoop running=True closed=False debug=False>
Traceback (most recent call last):
  File "C:\Users\Dell\anaconda3\envs\newDRL\lib\asyncio\windows_events.py", line 453, in finish_recv
    return ov.getresult()
OSError: [WinError 995] The I/O operation has been aborted because of either a thread exit or an application request

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Dell\anaconda3\envs\newDRL\lib\asyncio\proactor_events.py", line 768, in _loop_self_reading
    f.result()  # may raise
  File "C:\Users\Dell\anaconda3\envs\newDRL\lib\asyncio\windows_events.py", line 808, in _poll
    value = callback(transferred, key, ov)
  File "C:\Users\Dell\anaconda3\envs\newDRL\lib\asyncio\windows_events.py", line 457, in finish_recv
    raise ConnectionResetError(*exc.args)
ConnectionResetError: [WinError 995] The I/O operation has been aborted because of either a thread exit or an application request
C:\Users\Dell\anaconda3\envs\newDRL\lib\site-packages\jupyter_client\threaded.py:73: RuntimeWarning: ZMQStream only supports the base zmq.Socket class.

                Use zmq.Socket(shadow=other_socket)
                or `ctx.socket(zmq.DEALER, socket_class=zmq.Socket)`
                to create a base zmq.Socket object,
                no matter what other kind of socket your Context creates.
                
  self.stream = zmqstream.ZMQStream(self.socket, self.ioloop)
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.8.0
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.1 (OK)
chardet >=2.0.0               :  5.1.0 (OK)
cloudpickle >=0.5.0           :  2.2.1 (OK)
cookiecutter >=1.6.0          :  2.1.1 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.0.2 (OK)
IPython >=7.6.0               :  7.33.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  4.17.3 (OK)
keyring >=17.0.0              :  23.9.3 (OK)
nbconvert >=4.0               :  7.2.9 (OK)
numpydoc >=0.6.0              :  1.5.0 (OK)
paramiko >=2.4.0              :  3.0.0 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.4 (OK)
pygments >=2.0                :  2.14.0 (OK)
pylint >=1.0                  :  2.16.1 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.2 (OK)
qtawesome >=0.5.7             :  1.2.2 (OK)
qtconsole >=5.0.3             :  5.2.2 (OK)
qtpy >=1.5.0                  :  2.3.0 (OK)
rtree >=0.8.3                 :  1.0.1 (OK)
setuptools >=39.0.0           :  67.1.0 (OK)
sphinx >=0.6.6                :  5.1.1 (OK)
spyder_kernels >=2.0.1,<2.1.0 :  2.0.5 (OK)
textdistance >=4.2.0          :  4.5.0 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  25.0.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  3.6.3 (OK)
numpy >=1.7                   :  1.24.2 (OK)
pandas >=1.1.1                :  1.5.2 (OK)
scipy >=0.17.0                :  1.10.0 (OK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by dalthviz** [comment]
Hi @SyedaHinaAli thank you for the feedback! I think this problem is related with the Python version you are using (Python 3.8.0). Is possible for you to update Python (to for example Python 3.8.16)? This could be related to https://github.com/spyder-ide/spyder/issues/19505

**Comment by SyedaHinaAli** [comment]
ok thanks

**Comment by ccordoba12** [comment]
You also need to update Spyder to our latest version available in Anaconda (5.3.3). For that, please close Spyder, open the Anaconda Prompt and run there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).

**Comment by dalthviz** [comment]
Closing as a duplicate of https://github.com/spyder-ide/spyder/issues/19505
